android/app/src/main/java/org/fossasia/openevent/utils/SharedPreferencesUtil.java

```diff
@@ -0,0 +1,54 @@
+package org.fossasia.openevent.utils;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+import org.fossasia.openevent.OpenEventApp;
+
+public class SharedPreferencesUtil {
+    private static SharedPreferences sharedPreferences;
+    private static SharedPreferences.Editor editor;
```

Comment:
@iamareebjamal I'm not storing context here. I am making static SharedPreference object which is returned by context.getSharedPreferences() method. It should not create a memory leak. Right ?